__test__/components/sorter/Sorter.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Sorter from '../../../src/js/components/sorter/Sorter.jsx';
+import { render, mount } from 'enzyme';
+
+describe('Sorter', () => {
+  it('should rendered correctly', () => {
+    const component = render(<Sorter sortby={'DATE'} changeHandler={() => {}} />);
```

Comment:
Whereever you have a handler or simple function please use proper jest-way: https://jestjs.io/docs/en/mock-functions

you can write:
`const mockHandler = jest.fn()`
then use it
`changeHandler={mockHandler}`
and then you can check
`expect(mockHandler). toHaveBeenCalled();`
so you can check if handler was invoked and with which arguments or how many times: https://jestjs.io/docs/en/expect#tohavebeencalled

this applies to all tests where you're using `() => {}`